### PR TITLE
Add story about computer looking out of the window

### PR DESCRIPTION
**Pull request by PiotrBerebecki**
Please review @casey-collab

### CODE REVIEW

File: english/marshmallow.md [hunk 1]
```diff
@@ -996,3 +996,5 @@ You looking for a country for advanture? Try [Taiwan](trip-to-Taiwan/trip-to-Tai
 you find a mysterious cup with clear liquid in it. [drink?](drink/drink.md)
 
 Time to check out the latest [Total War - Warhammer](total-war/war-start.md)!
+
+You wake up in the morning and your [computer is looking out of the window](computer-windowd/computer-window.md)
```
Comment (casey-collab) [inline english/marshmallow.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/computer-window/computer-window.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+You wake up in the morning and your computer is looking out of the window
```
Comment (casey-collab) [inline english/computer-window/computer-window.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by PiotrBerebecki** [comment]
@casey-collab, have a look, it should be working OK now

**Comment by casey-collab** [comment]
Looks great! :smile:

**Pull request merged**